dashboard/app/helpers/levels_helper.rb

```diff
@@ -38,6 +38,17 @@ def create_channel(data = {}, src = nil)
     ))
     headers = result[1]
 
+    # The ChannelsApi may set a storage_id cookie that we need to copy to the response.
+    if headers['Set-Cookie']
+      cookie = headers['Set-Cookie'].split(';').map{ |token| token.strip.split('=') }
+      name = cookie[0][0]
+      cookie[0][0] = 'value'
+      cookie = Hash[cookie]
+      cookie['value'] = CGI.unescape cookie['value']
+      cookie['expires'] = Time.parse cookie['expires']
+      cookies[name] = cookie
```

Comment:
The clean way to do this would seem to be to call `StorageApps.create` directly: